src/test/mir-opt/matches_reduce_branches.rs

```diff
@@ -0,0 +1,12 @@
+// EMIT_MIR matches_reduce_branches.foo.MatchBranchSimplification.diff
```

Comment:
in order for CI to pass you need to commit the file generated by `--bless`, too

Reply:
ah yea, it was taking forever to run so I'll try to do that today